packages/webapp/src/components/ChartSection/Notices.jsx

```diff
@@ -20,14 +19,8 @@ const Text = styled(Typography)`
   }
 `;
 
-const callText = notices => notices.map(notice => (
-  <Text key={notice}>{notice}</Text>
-));
+const callText = notices => notices.map(notice => <Text key={notice}>{notice}</Text>);
 
-const Notices = ({ notices }) => (
-  <Wrapper>
-    {callText(notices)}
-  </Wrapper>
-);
+const Notices = ({ notices }) => <Wrapper>{callText(notices)}</Wrapper>;
```

Comment:
This seems like overzealous one-liners because we can.


Surely it'd be easier to understand what markup we're building here with

```jsx
const Notices = ({notices}) =>
    <Wrapper>
        notices.map(notice => <Text key={notice}>{notice}</Text>)
    </Wrapper>;
```

Reply:
like...the way it's written, if you want to know what Notices does, you're like _it's a wrapper, and now I need to go read that function over there_

what's Notices actually? It's a list of notices...